src/palette/internal/palettecreator.cpp

```diff
@@ -90,32 +90,51 @@ using namespace mu;
 using namespace mu::palette;
 using namespace mu::engraving;
 
-template<typename T> std::shared_ptr<T> makeElement(mu::engraving::Score* score)
-{
-    return std::make_shared<T>(score->dummy());
-}
+// Default wrapper...
+template<class C>
+struct makeElementImplWrapper {
+    template<typename ... Args>
+    static std::shared_ptr<C> makeElementImpl(mu::engraving::Score* score, Args&&... args)
```

Comment:
Ahh, I see now why you did it this way: I had forgotten that "Function template partial specialization is not allowed". Too bad... 
But this is a good alternative, well done!